addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -186,6 +189,36 @@ MockServer.include({
         });
         return messages;
     },
+    /**
+     * Simulate the 'message_post' Python method
+     *
+     * @private
+     * @return {Object[]}
+     */
+    _mockMessagePost: function (args) {
+        const {
+            args: [res_id],
+            model: res_model,
+            kwargs: postData,
+        } = args;
+        const model = this.data['mail.message'];
+        const modelFieldNames = Object.keys(model.fields);
+        const record = {};
+        for (const fieldName in postData) {
+            if (modelFieldNames.includes(fieldName)) {
+                record[fieldName] = postData[fieldName];
+            }
```

Comment:
which fields had an issue? If so, maybe these fields should be defined in `messaging.testUtils:beforeEach()`?

Reply:
None, I was thinking to filter all passed fields so that only those defined in data.fields are considered. You think it's better to take it all and let it fail if it's misused ?

Reply:
Yes, I prefer crash on undefined field, similarly to how unimplemented route make it crash

Reply:
Makes sense indeed, I'll change that